.env.common

```
@@ -0,0 +1,57 @@
+# This file contains configuration that is the same across all environments
+
+# === Application Settings ===
+NODE_ENV=production
+HOST=0.0.0.0
+
+# === Email Service Configuration ===
+EMAIL_SERVICE=smtp
+SMTP_HOST=mail.utia.cas.cz
+SMTP_PORT=465
+SMTP_SECURE=true
+SMTP_REQUIRE_TLS=false
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The SMTP_REQUIRE_TLS key should go before the SMTP_SECURE key

(UnorderedKey)